### PR TITLE
Fixes #25036: Include Ubuntu 24.04 in the list of supported OS

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/25036

### CODE REVIEW

File: src/reference/modules/installation/pages/operating_systems.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ available freely for these platforms:
 | OS | Version | Architecture
 
 | Debian | 11 and 12 | 64bit
-| Ubuntu | 20.04 LTS and 22.04 LTS | 64bit
+| Ubuntu | 20.04 LTS and 24.04 LTS | 64bit
```
Comment (ElaadF) [inline src/reference/modules/installation/pages/operating_systems.adoc:28]: Do we still support 20.04 for community ?

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Review by amousset — APPROVED** [review]

File: src/reference/modules/installation/pages/operating_systems.adoc [hunk 2]
```diff
@@ -105,7 +105,7 @@ The following operating systems are supported as a Rudder relay or server:
 | OS | Version | Architecture
 
 | Debian | 11 and 12 | 64bit
-| Ubuntu | 22.04 LTS | 64bit
+| Ubuntu | 22.04 LTS and 24.04 LTS | 64bit
```
Comment (amousset) [inline src/reference/modules/installation/pages/operating_systems.adoc:108]:
```suggestion
| Ubuntu | 22.04 LTS | 64bit
```

Only the agent support was added

**Comment by ElaadF** [comment]
OK, merging this PR

**Pull request merged**